build.gradle

```diff
@@ -148,26 +149,55 @@ tasks.register('checkAkitInstall', JavaExec) {
     classpath = sourceSets.main.runtimeClasspath
 }
 
-tasks.register('installLocalGitHookPostCheckout', Copy) {
-    from new File(rootProject.rootDir, '.scripts/post-checkout')
-    into { new File(rootProject.rootDir, '.git/hooks') }
-    filePermissions {
-        unix(0775)
-    }
+// Setup local git settings
+def gitConfig = 'Git Configuration'
+tasks.register('installLocalGitHookSymlinks') {
+    group = gitConfig
+
+    outputs.dir "$rootDir/.git/hooks/"
+
     doLast {
-        println("---Added Post Checkout Git Hook Script---")
+        def linkDir = file("$rootDir/.git/hooks/").toPath()
+        def gitHooks = fileTree("$rootDir/.scripts/git-hooks/")
+        gitHooks.forEach { File target ->
+            def link = linkDir.resolve(target.toPath().getFileName())
+
+            if (!Files.isSymbolicLink(link)) {
+                if (Files.exists(link)) {
+                    Files.delete(link)
+                }
+
+                def rel = { f -> rootDir.toPath().relativize(f) }
+                Files.createSymbolicLink(link, target.toPath())
+
+
+                println("---Installed Symbolic Link ${rel(link)} -> ${rel(target.toPath())} ---")
+            }
+        }
```

Comment:
so much line spaces, also give rel an informative name

Reply:
the name link and relativePath doesn't tell a lot to someone who didn't write this code. Try to name it with more informative names

Reply:
I've factored this out to a separate `ensureSymlink` function, I think the names are more fitting given that it's built for a more generic case